src/models/monitoringReviewGrantee.js

```diff
@@ -0,0 +1,93 @@
+const { Model } = require('sequelize');
+
+export default (sequelize, DataTypes) => {
+  class MonitoringReviewGrantee extends Model {
+    static associate(models) {
+      models.MonitoringReview.hasMany(
+        models.MonitoringReviewGrantee,
+        {
+          foreignKey: 'reviewId',
+          targetKey: 'reviewId',
+          as: 'monitoringReviewGrantees',
+        },
+      );
+
+      models.MonitoringReviewGrantee.belongsTo(
+        models.MonitoringReview,
+        {
+          foreignKey: 'reviewId',
+          targetKey: 'reviewId',
+          as: 'monitoringReview',
+        },
+      );
+
+      models.Grant.hasMany(
```

Comment:
I understand for sequencing reasons you suggest we make the associations in the file that demands the association, but it is extremely hard to work with the associations when they are all spread out over different files. Can we please add some comments to associated files for reference? Really what would be most helpful would just be copying this stuff over?

For example, in ```model/grant.js``` in the associate section, add: 

```js
/* from src/models/monitoringReviewGrantee.js
      models.Grant.hasMany(
        models.MonitoringReviewGrantee,
        {
          foreignKey: 'number',
          targetKey: 'grantNumber',
          as: 'monitoringReviewGrantees',
        },
      );
*/

Reply:
working on fixing now